### PR TITLE
MODLD-564: readme + env variables alignment

### PR DESCRIPTION
**Pull request by PBobylev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,62 +18,58 @@ Skip tests:
...
+| DB_PASSWORD                                         | postgres                        | Postgres password                                                                                                                                                                     |
+| DB_DATABASE                                         | postgres                        | Postgres database name                                                                                                                                                                |
+| SPRING_PROFILES_ACTIVE                              | -                               | Indicates if the application has to be run in FOLIO mode or standalone mode                                                                                                           |
+| KAFKA_HOST`*`                                       | kafka                           | Kafka broker hostname                                                                                                                                                                 |
```
Comment (pkjacob) [inline README.md:45]: Can we add `server.port` with default value `8081` here?

File: src/main/resources/application-folio.yml [hunk 2]
```diff
@@ -3,8 +3,8 @@ mod-linked-data:
     page-size: 1000
   kafka:
     topic:
-      work-search-index: linked-data.work
-      instance-ingress: inventory.instance_ingress
+      work-search-index: ${KAFKA_WORK_SEARCH_INDEX_TOPIC:linked-data.work}
+      instance-ingress: ${KAFKA_INVENTORY_INSTANCE_EVENT_TOPIC:inventory.instance_ingress}
```
Comment (pkjacob) [inline src/main/resources/application-folio.yml:7]: I think this should be KAFKA_INVENTORY_INSTANCE_INGRESS_EVENT_TOPIC to match with line 62 of readme file.

File: README.md [hunk 3]
```diff
@@ -85,124 +78,198 @@ curl --location '{{ base-uri }}/resource' \
...
+# Integration with FOLIO
+When running in FOLIO mode, this module integrates with multiple Folio modules via Kafka.
+## Search module
+The Linked Data module pushes a new and updated Work resource descriptions to the Kafka topic defined by KAFKA_WORK_SEARCH_INDEX_TOPIC env variable.
+The [mod-search](https://github.com/folio-org/mod-search) module consumes message and indexes a Work resource descriptions in [OpenSearch](https://aws.amazon.com/opensearch-service/) index for search and retrieval.
+## Source Record Storage module
+The [mod-source-record-storage](https://github.com/folio-org/mod-source-record-storage) module pushes Source Record domain events to the topic defined by KAFKA_SOURCE_RECORD_DOMAIN_EVENT_TOPIC_PATTERN env variable.
+The Linked Data module consumes a message and creates/updates a corresponding Authority resource in Linked Data graph.
+## Inventory module
+1. The Linked Data module pushes a new and updated Instance resource descriptions to the Kafka topic defined by KAFKA_INVENTORY_INSTANCE_EVENT_TOPIC_PATTERN env variable.
```
Comment (pkjacob) [inline README.md:271]: I think the env variable should be KAFKA_INVENTORY_INSTANCE_INGRESS_EVENT_TOPIC.

`KAFKA_INVENTORY_INSTANCE_EVENT_TOPIC_PATTERN` is used for the suppression flags (as mentioned in line 274)

**Comment by pkjacob** [comment]
Hi @PBobylev, Can we remove `SYSTEM_USER_PASSWORD` environment variable from Module descriptor? It is not needed for our application.

**Review by AndreiBordak — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (PBobylev) [reply README.md:45]: it's not an environment variable, is it?
But we could use a default one provided by Spring: [SERVER_PORT](https://docs.spring.io/spring-boot/how-to/webserver.html#howto.webserver.change-port)
Added, thank you.

File: src/main/resources/application-folio.yml (hunk 2 above)
Comment (PBobylev) [reply src/main/resources/application-folio.yml:7]: Yep, it's a mistake. Fixed, thanks!

**Review by askhat-abishev — APPROVED** [review]

**Comment by PBobylev** [comment]
> Hi @PBobylev, Can we remove `SYSTEM_USER_PASSWORD` environment variable from Module descriptor? It is not needed for our application.

Done and other env variables there were updated too.

**Review by pkjacob — APPROVED** [review]

**Pull request merged**